CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Additional logging of historical query flow in `UNSAFE` builds.
 - Historical query system will re-request entries if the host fails to provide them within a fixed time.
 - Renamed `ccf::CodeDigest` to `ccf:pal::PlatformAttestationMeasurement` and `get_code_id()` to `get_measurement()` (#5063).
+- Recovery procedure:
+  - The `GET /gov/recovery_share` endpoint has been replace with `GET /gov/recovery_share/{member_id}`. This can be accessed without any authentication.
+  - The `POST /gov/recovery_share` endpoint has been replaced with `POST /gov/recovery_share/{member_id}`. This request must now be signed, and will not allow purely TLS authentication.
+  - To sign a `POST /gov/recovery_share` request with COSE, the `ccf.gov.msg.type` protected header may now be `recovery_share`, and a new protected header `ccf.gov.msg.member_id` is now supported.
```

Comment:
member_id is redundant with kid

Reply:
Ah ha - I'd missed that it was already permitted server-side, because we didn't include it in the client-side `cose.py`. I'll clarify the CHANGELOG.

Reply:
`member_id` header removed, using existing `kid`.